packages/dev/s2-docs/src/Header.tsx

```diff
@@ -86,10 +78,20 @@ export default function Header(props: PageProps) {
               aria-controls={searchOpen ? searchMenuId : undefined}
               size="XL"
               isQuiet
-              onPress={openSearchMenu}
+              onPress={() => {
+                let library = getLibraryFromPage(currentPage);
+                let subdirectory = 's2';
+                if (library === 'internationalized' || library === 'react-aria') {
+                  // the internationalized library has no homepage so i've chosen to route it to the react aria homepage
+                  subdirectory = 'react-aria';
+
+                }
+                const url = new URL(`/${subdirectory}/index.html`, window.location.origin);
+                window.location.assign(url.href);
+              }}
```

Comment:
We should make this a link instead of a button so it's announced correctly and does navigation by default